types/block.go

```diff
@@ -2,12 +2,14 @@ package types
 
 type Block struct {
 	IdModel
-	Type                string            `json:"type"`
-	OrgName             string            `json:"orgName"`
-	StackName           string            `json:"stackName"`
-	Reference           string            `json:"reference"`
-	Name                string            `json:"name"`
-	ModuleSource        string            `json:"moduleSource"`
-	ModuleSourceVersion string            `json:"moduleSourceVersion"`
-	ParentBlocks        map[string]string `json:"parentBlocks"`
+	Type                string                     `json:"type"`
+	OrgName             string                     `json:"orgName"`
+	StackId             int64                      `json:"stackId"`
+	StackName           string                     `json:"stackName"`
```

Comment:
If this represents the data coming back from the furion API, I don't think StackName is returned anymore. We should either remove it here or if it is needed then make sure it is returned from furion.

Reply:
`stackName` is returned, but I will mark this as deprecated.